### PR TITLE
Update docs with research link

### PR DESCRIPTION
**Pull request by ashamis**

### CODE REVIEW

**Review by achamayou — APPROVED** [review]

File: doc/research.rst [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Glossary
+========
+
+`PAC: Practical Accountability for CCF <https://arxiv.org/abs/2105.13116>`_
+  Permissioned ledger systems execute transactions on a set of replicas governed by members of a consortium. They use Byzantine fault tolerance protocols to distribute trust among the replicas, and thus can ensure linearizability if fewer than 1/3 of the replicas misbehave. With more misbehaving replicas, current systems provide no guarantees, and all replicas and members share the blame.
```
Comment (achamayou) [inline doc/research.rst:5]: Should we have a "such as CCF" here?

File: doc/research.rst [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Glossary
```
Comment (achamayou) [inline doc/research.rst:1]:
```suggestion
Research
```

File: doc/research.rst (hunk 1 above)
Comment (achamayou) [reply doc/research.rst:5]: Also "Byzantine-fault tolerant" or perhaps "Byzantine-fault-tolerant" or something but not "tolerance" I would think?

File: doc/research.rst [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+  We describe PAC, a permissioned ledger system that assigns blame to misbehaving replicas while supporting governance transactions to change the consortium membership and the set of replicas. PAC signs and stores protocol messages in the ledger and provides clients with signed, universally-verifiable receipts as evidence that a transaction executed at a certain ledger position. If clients obtain a sequence of receipts that violate linearizability, anyone can audit the ledger and the sequence of receipts to assign blame to at least 1/3 of the replicas, even if all replicas and members misbehave. Auditing assigns blame by finding contradictory statements signed by the same replica. Since the set of replicas changes, PAC determines the valid signing keys at any point in the ledger using a shorter sub-ledger of governance transactions. PAC provides a strong disincentive to misbehavior at low cost: it can execute more than 48,000~transactions per second, and clients receive receipts in two network round trips.
```
Comment (achamayou) [inline doc/research.rst:7]: No tilde after the 48,000?

File: doc/research.rst [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+
+`CCF: A Framework for Building Confidential Verifiable Replicated Service <https://github.com/microsoft/CCF/blob/main/CCF-TECHNICAL-REPORT.pdf>`_
+  This paper present CCF, a framework to build premissioned confidential blockchains. CCF provides a simple programming model of a highly-available data store and a universally-verifiable log that implements a ledger abstraction. CCF leverages trust in a consortium of governing members and in a network of replicated hardware-protected execution environments to achieve high throughput, low latency, strong integrity and strong confidentiality for application data and code executing on the ledger.
```
Comment (achamayou) [inline doc/research.rst:11]:
```suggestion
  This paper present CCF, a framework to build permissioned confidential blockchains. CCF provides a simple programming model of a highly-available data store and a universally-verifiable log that implements a ledger abstraction. CCF leverages trust in a consortium of governing members and in a network of replicated hardware-protected execution environments to achieve high throughput, low latency, strong integrity and strong confidentiality for application data and code executing on the ledger.
```

**Review by achamayou — APPROVED** [review]

File: doc/research.rst [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+
+
+`CCF: A Framework for Building Confidential Verifiable Replicated Service <https://github.com/microsoft/CCF/blob/main/CCF-TECHNICAL-REPORT.pdf>`_
+  This paper present CCF, a framework to build permissioned confidential blockchains. CCF provides a simple programming model of a highly-available data store and a universally-verifiable log that implements a ledger abstraction. CCF leverages trust in a consortium of governing members and in a network of replicated hardware-protected execution environments to achieve high throughput, low latency, strong integrity and strong confidentiality for application data and code executing on the ledger.
```
Comment (eddyashton) [inline doc/research.rst:11]:
```suggestion
  This paper presents CCF, a framework to build permissioned confidential blockchains. CCF provides a simple programming model of a highly-available data store and a universally-verifiable log that implements a ledger abstraction. CCF leverages trust in a consortium of governing members and in a network of replicated hardware-protected execution environments to achieve high throughput, low latency, strong integrity and strong confidentiality for application data and code executing on the ledger.
```

**Review by eddyashton — APPROVED** [review]

File: doc/research.rst (hunk 1 above)
Comment (ashamis) [reply doc/research.rst:5]: This is verbatim from the paper's abstract so I would prefer to keep the two in sync, but we can change the paper text and when we iterate it we will update the text here.

**Review by jumaffre — APPROVED** [review]

**Pull request merged**